tests/feature_config.rs

```diff
@@ -1,14 +1,14 @@
 use assert_cmd::{assert::Assert, Command};
 
+static SUBCOMMAND_ARGS_INDEX: usize = 1;
```

Comment:
This is an implementation detail on `CargoSemverChecks`, so consider moving it into `impl CargoSemverChecks` instead. The invariant it corresponds to is the way the `args` value is constructed inside `CargoSemverChecks::new()`, so the constant should be near there as well.

Also, perhaps `const` instead of `static`? The difference is subtle and it's not a big deal either way.

Reply:
I think I cannot use a static inside the `impl` block, but changed it to const and moved it to the `impl CargoSemverChecks` :)